testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/settings/SettingsMenuActivity.kt

```diff
@@ -156,7 +158,9 @@ class SettingsMenuActivity : BaseActivity() {
 
         launch {
             try {
-                MiniApp.instance(AppSettings.instance.miniAppSettings).listMiniApp()
+                withContext(Dispatchers.IO) {
```

Comment:
the `launch` is already set with IO in `BaseActivity`.

Reply:
Ah yes, we can use that one from `BaseActivity`.
Sorry, I am going to close this PR as we don't need to set IO additionally here.